docs/specs/clients/notify/authentication.md

```diff
@@ -177,6 +177,136 @@ Paginated list of notifications with the most recently sent first. Unread notifi
 | Tag     | 4015         |
 | Topic   | notify topic |
 
+## `wc_notifyNotificationChanged`
+
+Emitted by the Notify Server when a notification changed state. For example if its read status changed.
+
+### Request
+
+- act - `notify_notification_changed`
+- iss - did:key of dapp authentication key
+- aud - did:key of client identity key
+- nfn - array of [Notify Notification](./data-structures.md#notify-notification)
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4018         |
+| Topic   | notify topic |
+
+### Response
+
+- act - `notify_notification_changed_response`
+- iss - did:key of client identity key
+- ksu - key server for identity key verification
+- aud - did:key of dapp authentication key
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4019         |
+| Topic   | notify topic |
+
+## `wc_notifyReadNotification`
+
+Marks a notification as read.
+
+### Request
+
+- act - `notify_read_notification`
+- iss - did:key of client identity key
+- ksu - key server for identity key verification
+- aud - did:key of dapp authentication key
+- app - did:web of app domain that this request is associated with 
+  - Example: `did:web:app.example.com`
+- ids - array of notification IDs to mark as read, max 1000 items
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4020         |
+| Topic   | notify topic |
+
+### Response
+
+- act - `notify_read_notification_response`
+- iss - did:key of dapp authentication key
+- aud - did:key of client identity key
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4021         |
+| Topic   | notify topic |
+
+## `wc_notifyGetUnreadNotificationsCount`
```

Comment:
Suggest removal in-favor of adding `last_sent_at` and `unread_count` to `NotifyServerSubscription`.

[Context](https://walletconnect.slack.com/archives/C044SKFKELR/p1704894912282769?thread_ts=1704891056.777619&cid=C044SKFKELR)